src/index.ts

```diff
@@ -247,8 +247,14 @@ export const release = {
 	addVirtualFieldsToModel: addReleaseAdditionsToModel,
 };
 export const deviceTypes = {
-	getAccessibleDeviceTypes,
-	findBySlug,
+	// TODO: Drop me in the next major
```

Comment:
The next major might be for example
See: https://github.com/balena-io/open-balena-api/pull/1607